server/src/main/java/org/elasticsearch/search/aggregations/support/ValuesSource.java

```diff
@@ -307,7 +367,7 @@ public boolean needsScores() {
 
             @Override
             public SortedBinaryDocValues bytesValues(LeafReaderContext context) throws IOException {
-                return new ValuesSource.WithScript.BytesValues(delegate.bytesValues(context), script.newInstance(context));
+                return new Bytes.WithScript.BytesValues(delegate.bytesValues(context), script.newInstance(context));
```

Comment:
@polyfractal As per above comment, this is one of the rough edges.  We're in a `Numeric` subclass, but we have this `bytesValues` method which calls into a different subclass now.  I'm just guessing, but I suspect this is why `WithScript` wasn't a `Bytes` subclass originally.